README.md

```diff
@@ -1,2 +1,3 @@
 # hello-world
-Chào cả nhà
+Em chào cả nhà.
+Năm mới vui vẻ
```

Comment:
Other reviewer says: Can you remove this last line please? I dont want them to have vui vẻ 😈